### DISCUSSION TITLE
How to get Avalonia version from code?

### DISCUSSION THREAD

**Discussion by sn4k3** [AvaloniaUI/Avalonia#6871] [Q&A] [answered]
I would like to show/report the avalonia framework version on my app.
Is there a variable i can access to do such?

Right now i'm using: `typeof(AvaloniaObject).Assembly.GetName().Version.ToString(3);`

**Comment by kekekeks** [comment, marked as answer]
`typeof(AvaloniaObject).Assembly.GetName().Version`
